docs/api-docs/cart-and-checkout/custom-order-confirmation.md

```diff
@@ -0,0 +1,91 @@
+* `checkoutId`: If present, the ID of the active checkout. You can't have both `orderId` and `checkoutId`
+
+To use the data exposed in the window object, include relevant code in your loader file. Here is an example of this code :
+
+```js
+ReactDOM.render(
+  <OrderConfirmation orderId={ window.checkoutConfig.orderId } />,
+  document.getElementById(window.checkoutConfig.containerId)
+);
+```
+
+## Hosting a custom order confirmation page
+
+You will need to host the custom confirmation file online so it can be served by the store. You can use a hosting service, such as [Amazon S3](https://aws.amazon.com/s3/). Using an external host will allow you the freedom to automate the build process if you wish and push updates automatically from your local machine.
+
+### Using WebDAV to host a custom order confirmation page
+
+You can upload a custom order confirmation page to your store's server using WebDAV. The instructions to upload the `/dist` folder to the BigCommerce server using WebDav are below:
+
+1. Before proceeding, ensure you have downloaded [Cyberduck](https://cyberduck.io/), our recommended WebDAV client.
+**Note:**  For more information on how to use Cyberduck, refer to the [File Access (WebDAV)](https://support.bigcommerce.com/s/article/File-Access-WebDAV) page.
+2. From your store control panel, navigate to **Server Settings** > **File Access (WebDAV)**.
+    - To automatically connect with Cyberduck, see [Single-click Login](https://support.bigcommerce.com/s/article/File-Access-WebDAV#login).
+    - To manually connect with Cyberduck, see [Connecting with Cyberduck Manually](https://support.bigcommerce.com/s/article/File-Access-WebDAV#manual).
+3. From Cyberduck, enter the `/content` folder and create a new folder named *OrderConfirmation*.
+4. Navigate into the `/dist` folder in your OrderConfirmation project.
+5. Copy the contents of the `/dist` folder and paste it into the *OrderConfirmation* folder.
+
+## Installing a custom order confirmation page  
+
+To install a custom order confirmation page on a store, follow these steps:
+
+1. Navigate to **Advanced Settings > Checkout** in your store's control panel.
+2. Under **Custom Order Confirmation**, type the following in the **Script URL** field, replacing <version> with the latest version number generated by running the `npm run release:alpha` command:
+  ```
+  `webdav:checkout/auto-loader-<version>.js`
+  ```
+  NOTES:
+  
+  *Prepending `webdav:` Indicates that the URL is in the remote WebDAV directory. It will treat `/content` as the root WebDAV directory.
+  
+  *It is important to include `<version>` number in the Script URL field. Because if you make changes to the same provided loader filename, you could serve a cached version to the user.
+  
+![order-confirmation-page](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/order-confirmation-page.png "Custom Order Confirmation Page")
```

Comment:
Recommend putting this image right next to the step it pertains to. The Note section should follow it.